cookbook/add-tabs.rst

```diff
@@ -0,0 +1,300 @@
+Adding tabs to Sulu admin interface
```

Comment:
In order to better match other headings I would rephrase it to "Adding tabs to Sulu's Admin UI"